example/windows/.gitignore

```
@@ -0,0 +1,24 @@
+.vs/
+
+# User-specific files
+*.suo
+*.user
+*.sln.docstates
+
+# Build results
+[Dd]ebug/
+[Dd]ebugPublic/
+[Rr]elease/
+[Rr]eleases/
+AppPackages/
+x64/
```

Should we also ignore `ARM`?
```suggestion
AppPackages/
ARM/
x64/
```